frontend-react/src/pages/home/Sections/LiveMapSection.module.scss

```diff
@@ -0,0 +1,18 @@
+@use "../../../global-modules.scss" as *;
+
+.legend {
+    list-style: none;
+
+    li {
+        float: left;
+        margin-right: 8px;
```

Comment:
I might be late to this but do we typically use `px` units or a more fluid unit like `rem`?

Reply:
px is what we have been using